modules/admin_manual/pages/_partials/maintenance/major_release_note.adoc

```diff
@@ -0,0 +1,6 @@
+[TIP]
+====
+If required, you can skip major releases when upgrading your ownCloud installation.
+However, we recommend that you first upgrade to the latest point release of your respective minor version, e.g. _8.2.11_.
```

Comment:
Sorry, late. https://owncloud.org/news/owncloud-10-0-3-release/ said:

> It is now possible to directly upgrade from 8.2.11 to 10.0.3 in a single upgrade process

and https://github.com/owncloud/core/blob/v10.2.0/version.php#L33:

> $OC_VersionCanBeUpgradedFrom = [[8, 2, 11],[9, 0, 9],[9, 1]];

So example:

8.2.11 -> Directly 10.2.0
8.2.10 -> Need 8.2.11, then 10.2.0
9.0.9 -> Directly 10.2.0
9.0.8 -> Need 9.0.9, then 10.2.0

and older:

7.0.15 -> Need later 8.0.x, later 8.1.x, 8.2.1 then directly 10.2.0

Reply:
Done https://github.com/owncloud/docs/issues/1751